package.xml

```diff
@@ -8,10 +8,17 @@
 
   <license>MIT</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>mpi_cmake_modules</build_depend>
+
   <depend>libopencv-dev</depend>
-  <depend>pybind11_catkin</depend>
+  <depend>pybind11_vendor</depend>
+  <depend>pybind11</depend>
+
+  <test_depend>ament_cmake_nose</test_depend>
```

Comment:
If you want you can also use ament_cmake_pytest for using pytest.
I am not sure which one is better.
I know pytest is the default tool of colcon.

Reply:
pytest is currently throwing some error in my Singularity container and I anyway always use nosetest when running tests manually.
I assume it doesn't make a relevant difference.